src/app/containers/ContactsContainer.js

```diff
@@ -50,6 +51,9 @@ const ContactsContainer = ({ location, history }) => {
     const [user] = useUser();
     const [userKeysList, loadingUserKeys] = useUserKeys(user);
 
+    const { isMobile, isDesktop } = useActiveBreakpoint();
+    const noHeader = isMobile ? '--noHeader' : '';
```

Comment:
@econdepe I'm a bit confuse, you told me you want to handle `tablet` and `mobile` in the same way so why are you only using `isMobile` here?

Reply:
It's a little tricky:
- Unmounting the contacts list or the contact view (depending on whether a `contactID` is selected or not) is handled in the same way for tablet and mobile (we have to because having both components mounted looks bad on the tablet).
- The burger and other reponsive stuff appears only for mobile, not for tablet.

So certain things are specific for mobile, others are shared through tablet and mobile